html/includes/graphs/application/postfix_deferral.inc.php

```diff
@@ -0,0 +1,37 @@
+<?php
+require 'includes/graphs/common.inc.php';
```

Comment:
Could you remove `require 'includes/graphs/common.inc.php';` from each of the graph files and see if it's ok still. This is already included so doesn't need to be here.